### ISSUE TITLE
asphalt

### ISSUE THREAD

**Issue by ddorstijn** [CodeArtemis/TriggerRally#41]
1. One of the first things you should do is making a mew type of ground: Asphalt. just an simple option with the only difference of dirt is that asphalt is completely flatten. I dont think it is that hard to make just a different texture not?
2. I very nice idee would be to add signs where the creator can put a little text like 'Pitstop ->' or the name of the creator 'Jareiko'. won't that be awesome.
3. Just a little kind of a bug. when you go forward wikthout giving gas(when you are on a slope) the motor makes the same noiise as if you are giving gas. just a little bit weird.
4. A way to look behind you. On the moment i don't think there is a way and that is really frustating.

Help it is possible to realisable and not to hard.

Succes

**Comment by jareiko** [comment]
1. Adding different surface types is not as easy as it might seem, because the terrain system is quite simple. You can already make the dirt completely flat by choosing the maximum "dirt amount" value and a large "dirt shape" value.
2. Hmm, I will think about how I can do that.
3. Yes, the car engine noise is oversimplified. When you open the throttle, it just increases in volume rather than actually changing the quality of the sound. I need to record some new engine sounds to make it more realistic.
4. Good idea, and probably not too hard. Will add to the to-do list.

Thanks again for the suggestions! :smiley:

**Comment by copiousopioids** [comment]
I actually like how you did the sounds with raising the pitch at different levels. It almost sounds realistic at times.

If I remember right, in V1 you can look to the sides with < and > keys. That would be really cool to have.

**Comment by jareiko** [comment]
Yes, being able to look side to side would be useful! I'll add that to the to-do list as well, thanks.
